### DISCUSSION TITLE
Selecting multiple clips in the toolbox

### DISCUSSION THREAD

**Discussion by tomwolsky** [latenitefilms/BRAWToolbox#31] [Feature Requests]
Would it be possible to Cmd click or Shift click multiple clips in the toolbox so they can be removed? Could the Delete key be used to Remove Clips from the toolbox?

**Comment by tomwolsky** [comment]
If you remove a clip from the toolbox, could the next clip be automatically selected, the way the Finder works?

**Reply by latenitefilms** [reply]
Yes, this should be possible. Added to issue #38.

**Comment by tomwolsky** [comment]
If you have multiple clips selected in the toolbox, would it be possible to have the green drag icon simply drag the selected items into the library.

**Reply by latenitefilms** [reply]
Thanks for your detailed feedback @tomwolsky !

Regarding:

> If you have multiple clips selected in the toolbox, would it be possible to have the green drag icon simply drag the selected items into the library.

...and:

> Would it be possible to Cmd click or Shift click multiple clips in the toolbox so they can be removed?

The reason I haven't allowed the selection of multiple BRAW clips is that it gets insanely complicated in terms of what gets displayed in the Inspector, because each BRAW file can have different "read only" parameters, and parameter ranges. This basically falls in the "too hard" basket for now.

> Could the Delete key be used to Remove Clips from the toolbox?

Yes, this should be possible. Added to issue #37.

**Comment by tomwolsky** [comment]
As the only place I can see the clips is in the toolbox the software needs to have the ability to work more like the Finder. I would really like to have a right-click real in Finder function. I have about a 50 clips in the toolbox. I'm going through them an selecting to import what I need and what I can dump from the system.

**Reply by latenitefilms** [reply]
> I would really like to have a right-click real in Finder function

I assume you mean "Reveal In Finder"? Added to issue #39.

Keep in mind you can also drag and drop BRAW files from Finder to the Workflow Extension, so you could do the sorting at the Finder level first.

**Comment by tomwolsky** [comment]
Thanks Chris. I guess I'd have to do it one at a time. The numeric string names make it difficult to see what's what.
